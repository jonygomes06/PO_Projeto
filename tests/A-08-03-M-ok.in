5
1
Ani
ani@somewhere.com
2
4
3
1
Rey
rey@rey.org
3
1
Han Solo
han@star.war
3
0
0
