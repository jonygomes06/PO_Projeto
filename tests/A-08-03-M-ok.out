Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 4.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o número de utente: 4 - Ani - ani@somewhere.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 4 - Ani - ani@somewhere.com - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 5.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 4 - Ani - ani@somewhere.com - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
5 - Rey - rey@rey.org - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 6.
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 4 - Ani - ani@somewhere.com - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.com - NORMAL - ACTIVO
6 - Han Solo - han@star.war - NORMAL - ACTIVO
2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Obi-Wan Kenobi - obiwan@jedi.org - NORMAL - ACTIVO
5 - Rey - rey@rey.org - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 